lib/errors/course_update_error_handling.rb

```diff
@@ -1,13 +1,24 @@
 # frozen_string_literal: true
 
 module CourseUpdateErrorHandling
+  def perform_error_handling(error_record)
+    report_course_exception_sentry(error_record)
+    update_course_flags
+  end
+
   def report_course_exception_sentry(error_record)
     Raven.capture_exception(error_record.error,
                             level: error_record.level,
                             extra: error_record.sentry_extra,
                             tags: {
-                              course_update_id: error_record.sentry_tag_uuid,
-                              course: error_record.course_slug
+                              course_update_id: sentry_tag_uuid,
+                              course: @course.slug
                             })
   end
+
+  def update_course_flags
+    @course.flags[:errors] ||= {}
+    @course.flags[:errors][sentry_tag_uuid] ||= 0
+    @course.flags[:errors][sentry_tag_uuid] += 1
```

Comment:
I think we need to log the error counts and the sentry uuid as part of the update logging we already do (including only keeping the last 10 updates) instead of putting it in a separate `errors` key.

Reply:
Okay, I forgot about that 😅